docs/install/_linux.rst

```diff
@@ -37,18 +39,81 @@ The advantages of downloading a ``.deb`` file is the portability: you can copy t
 
        kolibri start
 	
-#. When command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
+#. When the command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
 
 
 Uninstall
----------
+*********
 
 * Open **Software** on Ubuntu and locate the Kolibri. Press **Remove**.
-* Or from command line: ``sudo apt-get remove kolibri``.
+* Or from the command line: ``sudo apt-get remove kolibri``.
+
 
 Upgrade
--------
+*******
 
 When you use the PPA installation method, upgrades to newer versions will be automatic, provided there is internet access available.
 
 To upgrade Kolibri on a Debian device without internet access, bring the updated ``.deb`` file and follow the same steps as in :ref:`lin_deb`.
+
+
+.. _rpi:
+
+Raspberry Pi (Raspbian)
+-----------------------
+
+Kolibri currently doesn't work out of the box for Raspbian Jessie. We are refining our distribution to work out of the box, but you need to follow these two steps in order to install it (tested on Kolibri 0.9):
+
+1. Running ``add-apt-repository`` as shown in the PPA instructions does not work. Instead, run::
+
+      echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list
+      sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys DC5BAA93F9E4AE4F0411F97C74F88ADB3194DD81
+      sudo apt update
+      sudo apt install kolibri
+
+2. Kolibri does not start after installation. This is because ``python3-cffi`` is outdated on Raspbian. Upgrade it like this::
+
+      sudo apt install libffi-dev
+      sudo pip3 install pip --upgrade
+      sudo pip3 install cffi --upgrade
+      sudo systemctl start kolibri
+
+3. When the command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
+
+.. warning: Loading channels can take a **long time** on a Raspberry Pi. When generating channel contens for Khan Academy, * Generating channel listing. This could take a few minutes…* means ~30 minutes. The device's computation power is the bottleneck. You might get logged out while waiting, but this is harmless and the process will continue. Sit tight!
+
+
+Troubleshooting
+^^^^^^^^^^^^^^^
```

Comment:
Recommend making this an 'info' block so that it doesn't add noise to the TOC:

![image](https://user-images.githubusercontent.com/2367265/38648214-dd2fe9b0-3da4-11e8-8651-8b6260532fd0.png)

Reply:
There's an intention to create more sections from my side, possibly turning this into a "How-to" instead.

Reply:
Might be best to break that into a separate file then so the top-level page stays simpler